### PR TITLE
Services: DataBase Model

### PR DESCRIPTION
**Pull request by MoritzHahn1337**

### CODE REVIEW

File: datameta/models/db.py [hunk 1]
```diff
@@ -78,6 +84,9 @@ class User(Base):
     files            = relationship('File', back_populates='user')
     passwordtokens   = relationship('PasswordToken', back_populates='user')
     apikeys          = relationship('ApiKey', back_populates='user')
+    services         = relationship("Service",
```
Comment (lkuchenb) [inline datameta/models/db.py:87]:
```suggestion
    services         = relationship('Service',
```

**Comment by MoritzHahn1337** [comment]
- [ ] #362 
- [ ] #364
- [ ] #366
- [ ] #367

File: datameta/models/db.py [hunk 2]
```diff
@@ -206,3 +216,22 @@ class ApplicationSettings(Base):
...
     time_value   = Column(Time, nullable=True)
+
+class Service(Base):
+    __tablename__ = "services"
+    id           = Column(Integer, primary_key=True)
+    uuid         = Column(UUID(as_uuid=True), unique=True, default=uuid.uuid4, nullable=False)
+    site_id      = Column(String(50), unique=True, nullable=False, index=True)
+    name         = Column(Text, nullable=True)
+    # Relationships
+    users        = relationship("User",
```
Comment (lkuchenb) [inline datameta/models/db.py:227]:
```suggestion
    users        = relationship('User',
```

File: datameta/models/db.py [hunk 3]
```diff
@@ -206,3 +216,22 @@ class ApplicationSettings(Base):
...
+                    secondary=user_service_table,
+                    back_populates="services")
```
Comment (lkuchenb) [inline datameta/models/db.py:229]:
```suggestion
                    back_populates='services')
```

File: datameta/models/db.py [hunk 4]
```diff
@@ -78,6 +84,9 @@ class User(Base):
     files            = relationship('File', back_populates='user')
     passwordtokens   = relationship('PasswordToken', back_populates='user')
     apikeys          = relationship('ApiKey', back_populates='user')
+    services         = relationship("Service",
+                    secondary=user_service_table,
+                    back_populates="users")
```
Comment (lkuchenb) [inline datameta/models/db.py:89]:
```suggestion
                    back_populates='users')
```

**Review by lkuchenb — APPROVED** [review]
Thanks, just the tick style is inconsistent with the rest of the file, otherwise good to go 👍

**Pull request merged**